### PR TITLE
Move TAP 8 to accepted

### PR DESCRIPTION
**Pull request by mnm678**

### CODE REVIEW

**Review by jkjell — COMMENTED** [review]
Looks good. I'm not sure if an optional/conditional client workflow is typically documented but, I asked for my own knowledge at least. 😅

**Review by lukpueh — DISMISSED** [review]
Looks good.

I must say though that I don't quite understand the sections about TAP 4 and TAP 3: Half of the TAP 4 section talks about mirrors, which are not related to TAP 4, and the other half is a bit vague ("repository manager must ensure that they have the same set of trusted keys after all rotations" ... who are they and why?).  And the section about TAP 3 seems to describe the same rotation process as without TAP 3. Or am I missing something?

Either way, I don't think these two sections should block the TAP. The basic idea sounds reasonable to me.

Unfortunately, the POC seems [outdated](https://github.com/theupdateframework/python-tuf/pull/2257#issuecomment-1715651345), but IIUC the official (and lived) TAP process does not required a full implementation before the final status.

File: tap8.md [hunk 1]
```diff
@@ -219,16 +205,13 @@ The client will then look for all files that begin with `rotate/foo.rotate` in
 the snapshot metadata. The client will process these in version order (ie starting
 with `rotate/foo.rotate.1`, then `rotate/foo.rotate.2` by first checking for this
 version file in a local cache. The client will then fetch the rotate file from
-remote. If the remote file is a rotation to null, and is signed with the currently
-trusted keys, the client will halt the verification of this metadata and act as if
-it is unverified when continuing the update process (and look for metadata from
-the next role in the pre-order depth-first search). Otherwise, the client should
+remote. The client should then
```
[the remark this answers is not in the document]
Comment (mnm678) [reply tap8.md:208]: Yeah, this provides similar protection if that optional hash is not included (basically a local cache instead of a snapshot cache).

**Comment by mnm678** [comment]
Thanks for the reviews @lukpueh @jkjell! I pushed a suggested change, which dismissed the reviews. If you could re-approve this should be good to go.

@lukpueh The TAP 3 process is basically the same, that section is just describing the compatibility.

**Review by lukpueh — APPROVED** [review]

**Review by jkjell — APPROVED** [review]

**Comment by trishankatdatadog** [comment]
Thanks all for moving this long-outstanding TAP fwd.

I can't speak for @jku, but we have some reservations about the complexity this TAP adds, although we understand the value it could add to OSS package registries like PyPI, so we probably weren't the best people to review it.

Thanks again!

**Pull request merged**